profile.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: profiles.fluxcd.io/v1alpha1
+kind: Profile
+metadata:
+  name: nginx
+spec:
+  description: Profile for deploying nginx
+  maturity: alpha
+  owner: WeaveWorks <gitops@weave.works>
```

Comment:
I suspect we could drop the `maturity` and `owner` from here _for now_?

Reply:
👍